core/src/main/java/org/elasticsearch/index/query/functionscore/ScoreFunctionBuilders.java

```diff
@@ -93,4 +93,16 @@ public static WeightBuilder weightFactorFunction(float weight) {
     public static FieldValueFactorFunctionBuilder fieldValueFactorFunction(String fieldName) {
         return new FieldValueFactorFunctionBuilder(fieldName);
     }
+
+    public static void registerBuilders(SearchModule searchModule, NamedWriteableRegistry namedWriteableRegistry) {
```

Comment:
This feels a little out of place, as this class is here for the java api. I guess there was no better place to put it?